src/content/doc-surrealql/statements/access.mdx

```diff
@@ -0,0 +1,481 @@
+---
+sidebar_position: 0
+sidebar_label: ACCESS 
+title: ACCESS statement | SurrealQL
+description: The ACCESS statement can be used to manage access grants.
+---
+
+import Since from '@components/shared/Since.astro'
+
+# `ACCESS` statement
+
+<Since v="v2.2.0" />
+
+> [!CAUTION]
+> Currently, the `ACCESS` statement is an experimental feature intended to be used for validating its suitability and security. As such, it may be subject to breaking changes and may present unidentified security issues. Do not rely on this feature in production applications. To enable this and other experimental features related to bearer access, set the `SURREAL_EXPERIMENTAL_BEARER_ACCESS` environment variable to `true`.
```

Comment:
```suggestion
> Currently, the `ACCESS` statement is an experimental feature intended to be used for validating its suitability and security. As such, it may be subject to breaking changes and may present unidentified security issues. Do not rely on this feature in production applications. To enable this and other experimental features related to bearer access, set the `SURREAL_EXPERIMENTAL_BEARER_ACCESS` [environment variable](/docs/surrealdb/cli/env) to `true`.
```